style.css

```
@@ -0,0 +1,259 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family:Arial, Helvetica, sans-serif;
+}
+section{
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
+    background: #112d42;
+}
+section::before{
+    content: '';
+    position: absolute;
+    inset: 0;
+    width: 50%;
+    height: 100%;
+    background: #03a9f4;
+}
+section .container{
+    position: relative;
+    min-width: 1100px;
+    min-height: 550px;
+    display: flex;
+    z-index: 5;
+}
+section .container .contactinfo{
+    position: absolute;
+    top: 40px;
+    width: 350px;
+    height: calc(100% - 80px);
+    background: #0f3959;
+    z-index: 1;
+    padding: 40px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    box-shadow: 0 20px 20px rgba(53, 5, 5, 0.2);
+}
+section .container .contactinfo h2{
+    color: #fff;
+    font-size: 24px;
+    font-weight: 500;
+}
+section .container .contactinfo .info{
+    position: relative;
+    margin: 20px 0;
+}
+section .container .contactinfo .info li{
+    position: relative;
+    
+    margin: 20px 0;
+    
+    cursor: pointer;
+    
+    list-style: none; 
+   
+    display: flex;
+    align-items: flex-start;
+    cursor: pointer;
+    list-style: none;
+    display: flex;
+    margin: 20px 0;
+    align-items: flex-start;
+}
+section .container .contactinfo .info li span:nth-child(1){
+    width: 30px;
+    min-width: 30px;
+}
+section .container .contactinfo .info li span:nth-child(1) img{
+    max-width: 100%;
+    filter: invert(1);
+    opacity: 0.5;
+}
+section .container .contactinfo .info li span:nth-child(2){   
+    color: #fff;
+    margin-left: 10px;
+    font-weight: 300;
+    opacity: 0.7;
+}
+section .container .contactinfo .info li:hover span:nth-child(1) img,
+section .container .contactinfo .info li:hover span:nth-child(2){
+    opacity: 1;
+}
+section .container .contactinfo .sci{
+    position: relative;
+    display: flex;
+}
+section .container .contactinfo .sci li{
+    list-style: none;
+    margin-right: 15px;
+}
+section .container .contactinfo ul.sci li a{
+    text-decoration: none;
+}
+section .container .contactinfo .sci li a img{
+    filter: invert(1);
+    opacity: 0.5;
+}
+section .container .contactinfo .sci li:hover a img{
+    opacity: 1;
+}
+section .container .contactform{
+    position: absolute;
+    padding: 70px 50px;
+    background: #fff;
+    margin: 0 0 0 150px;
+    padding: 0 0 0 250px;
```

```suggestion
    padding: 0 0 0 200px;
```
suggestion for the same issue this is change number-2